docs/permissions/plugin-authors/02-adding-a-basic-permission-check.md

```diff
@@ -235,7 +235,7 @@ Then we want to update the `plugins/todo-list-backend/src/service/standaloneServ
 +   ServerTokenManager,
   } from '@backstage/backend-common';
   import { DefaultIdentityClient } from '@backstage/plugin-auth-node';
-  import { ServerPermissionClient } from '@backstage/plugin-permission-node';
++ import { ServerPermissionClient } from '@backstage/plugin-permission-node';
```

Comment:
This line isn't present in the standard default permission policy from the docs, so needs a `+` prefix.

Reply:
Yup, another good catch, missed when I went though this last time 👍